src/metax_api/tests/models/catalog_record.py

```diff
@@ -120,7 +121,8 @@ def test_get_using_dict_error_not_found_1(self):
         self.assertEqual(found, False, 'get with using_dict should have not returned a result')
 
     def test_get_using_dict_error_preferred_identifier_not_allowed(self):
-        row = {'research_dataset': {'preferred_identifier': 'pid:urn:cr1'}, 'other_stuff': 'doesnt matter'}
+        row = {'research_dataset': {'preferred_identifier': 'urn:nbn:fi:att:5e0f691d-bf41-41b1-b86b-0b27a2a9f251'},
```

Comment:
Why is the identifier hard corded here, instead of using something from the db, like above?